index.html

```diff
@@ -40,6 +52,21 @@ <h2>Add new book</h2>
           <button type="submit" id="button">Add</button>
         </form>
       </div>
+      <!-- Contact Section -->
+      <div id="contact" class="contact-us">
+        <h2>Contact Information</h2>
+        <p>
+          Do have any question or you just want to say "hello" <br />
+          You can Reach out to us!
+        </p>
+        <ul>
+          <li>Our E-mail: books@awesome.com</li>
+          <li>Our Phone Number: 003311223344</li>
+          <li>Our Adress: John Doe street, 22, 883322 Vikings, Valhala</li>
+        </ul>
+      </div>
+      <!-- Footer Section -->
+      <footer><p>Copyright: All rights reserved</p></footer>
```

Comment:
- [ ] Kindly ensure that the text `Copyright`, and is aligned to the left of the _footer_. of each page.

**Your sample page:**
![image](https://user-images.githubusercontent.com/12745474/153090833-16883202-da39-4ea7-9c1c-8d86c646d7f2.png)


**The wireframe pages:**
![image](https://user-images.githubusercontent.com/12745474/138343768-db7df029-2766-4f6c-b59a-67fd2ac3deb3.png)